src/jenkinsgithublander/jobs.py

```diff
@@ -104,6 +119,7 @@ def mark_pull_request_build_failed(job_name, pr, build_number, failure_message,
         config['jenkins.merge.url'],
```

Comment:
Later generate_build_url uses two parts from the named tuple.  So I figured it was easier to just set the auth part to None rather than changing the signature here to pass the things actually needed.